setup.py

```diff
@@ -184,6 +184,9 @@ def get_model_zoo_configs() -> List[str]:
         "omegaconf>=2.1",
         "hydra-core>=1.1",
         "black==22.3.0",
+        "packaging",
+        # Lock version of protobuf because > 4.x breaks build
+        "protobuf<4.0",
```

Comment:
It is used once because once in this PR,

> This is needed by ONNX graph, which are internally represented through protobuf's proto. Both ONNX-only and Caffe2 ONNX need this

Not needed anymore, protobuf released a new minor version to fix the regression. Removing this dependency